ligo/README.md

```diff
@@ -1,14 +1,43 @@
-# BaseDAO LIGO implementation
+# BaseDAO LIGO
+
+__Note: This is still current in development.__
+
+This is the LIGO implementation of BaseDAO.
+
+The main difference between the two are:
+- `BaseDAO Lorentz`:
+    - uses compile-time configuration to configure various options of baseDAO, which means that each contract that uses `BaseDAO Lorentz` will have its own `michelson` contract.
+
+- `BaseDAO LIGO`: 
+    - uses runtime configuration
+    - only one `michelson` contract is used, the user simply defines the configuration as part of originated storage
 
 ## Prerequisites
 
 You will need [ligo executable](https://ligolang.org/docs/intro/installation) installed.
 
 ## Build instructions
 
-Use Makefile to build a contract, for instance
-
+To build the LIGO contract, simply run:
 ```sh
 make all
 ```
 
+## Generating `Storage`
+TODO
+
+## Originating the contract
+To originate `baseDAO LIGO`, run:
+```
+TODO
```

Comment:
@pasqu4le Do we have a sample storage for LIGO dao? Since to originate the contract, we need a storage.

Hope to get review on the rest of the change as well.

Reply:
I'm not sure we'll end up providing one at all, hence why the storage section is a TODO for now.

I suggested below in a comment how I think we should handle this.